e2etest/pkg/frr/bgp.go

```diff
@@ -111,3 +111,14 @@ func RawDump(exec executor.Executor, filesToDump ...string) (string, error) {
 
 	return res, nil
 }
+
+func CommunityInfo(exec executor.Executor, community string) error {
```

Comment:
I'd rename this to "ContainsCommunity" since it does not return the info.
Also, can you add the doc text?